website/docusaurus.config.js

```diff
@@ -856,6 +856,10 @@ module.exports = {
                         ],
                         to: '/topics/feature-flag-migration/feature-flag-migration-best-practices',
                     },
+                    {
```

Comment:
@nnennandukwe for future reference, docs to docs redirects can be done in this file

Reply:
Actually also added it in vercel, not sure if we need both

Reply:
I think in the future we probably want to move every redirect to Vercel. But I'm not sure what the limit is 😅